packages/oidc-dpop-client-browser/rollup.config.js

```diff
@@ -0,0 +1,32 @@
+import pkg from "./package.json";
+import typescript from "rollup-plugin-typescript2";
+
+export default {
+  input: "./src/index.ts",
+  output: [
+    {
+      file: pkg.main,
+      format: "cjs",
+    },
+    {
+      file: pkg.module,
+      format: "esm",
+    },
+    {
+      dir: "umd",
+      format: "umd",
+      name: "LitSolidCore",
```

Comment:
```suggestion
      name: "OidcDpopClient",
```

Reply:
That's not the first time this happens ^^